docpages/02_disdppgloss.md

```diff
@@ -1,51 +1,51 @@
+* **Action row**: A collection of up to five **components** which is attached to a message.
 
-2. **Audit log**: A log of **events** that have happened in a **guild**.
+* **Audit log**: A log of **events** that have happened in a **guild**.
 
-3. **Auto mod**: Discord's low-code solution to moderation. However, it is very limited in scope.
+* **Auto mod**: Discord's low-code solution to moderation. However, it is very limited in scope.
 
-4. **Badge**: A decoration on someone's profile showing certain things about them, such as if they have nitro, if they are a discord developer, etc.
+* **Badge**: A decoration on someone's profile showing certain things about them, such as if they have Nitro, if they are a Discord developer, etc.
 
-5. **Bot token**: A secret string of characters that is used as a "login" to your bot. If you lose it or it gets leaked you will have to get a new one from the discord developer portal, so be sure to keep it in a place that is both secure and where you won't forget it.
+* **Bot token**: A secret string of characters that is used as a "login" to your bot. If you lose it or it gets leaked you will have to get a new one from the Discord developer portal, so be sure to keep it in a place that is both secure and where you won't forget it.
 
-6. **Button**: A **component** on a message that can be styled that sends an **event** when clicked on by a user.
+* **Button**: A **component** on a message that can be styled that sends an **event** when clicked on by a user.
 
-7. **Cache**: A type of storage efficient for things like messages.
+* **Cache**: A type of storage efficient for things like messages.
 
-8. **Callback**: While not strictly related to discord, it is used a LOT in D++. A callback is when a function is passed to another function, sort of like how you might give someone a telephone number (you give them the means to do some sort of interaction rather than asking them how to interact), which is used to handle responses to **events**.
+* **Callback**: While not strictly related to Discord, it is used a LOT in D++. A callback is when a function is passed to another function, sort of like how you might give someone a telephone number (you give them the means to do some sort of interaction rather than asking them how to interact), which is used to handle responses to **events**.
 
-9. **Cluster**: A singular bot application, which is composed of one or more **shards**, a **cluster** is the center of bot development.
+* **Cluster**: A singular bot application, which is composed of one or more **shards**, a **cluster** is the centre of bot development.
 
-10. **\(Slash\) command**: The primary way a user interacts with a bot. It is a command sent to the bot with **parameters** (which may be optional) and is initiated by staring a message with `/`.
+* **Component**: A component is anything that can appear in a bot's message besides text, such as **buttons** and **select menus**.
```

Comment:
technically not everything - an embed, a file or an image are not a component

Reply:
I didn't change the text here, the Slash command just got moved down the list due to it now sorting under S and not C. Happy to change the text as a fix to this PR or separately though. One of the mentioned "other things that could use work".

Reply:
yeah this is a new suggestion not a fix